requirements.txt

```diff
@@ -19,3 +19,5 @@ numba==0.52
 umap-learn==0.4.6
 unidecode==0.4.20
 coqpit
+mecab-python3
+unidic-lite
```

Comment:
why `unidic-lite` ?

Reply:
unidic-lite is the Japanese dictionary used by MeCab. MeCab is a Japanese tokenizer which predicts phonemes from texts.